app/src/test/java/com/jetbrains/handson/mpp/mobile/TimeHelperTest.kt

```diff
@@ -0,0 +1,115 @@
+package com.jetbrains.handson.mpp.mobile
+
+import org.junit.Assert
+import org.junit.Test
+
+class TimeHelperTest {
+
+
+    @Test
+    fun getJourneyTime_isCorrect() {
+        val departure_time = "2022-07-24T14:34:00.000+01:00"
+        val arrival_time = "2022-07-24T15:35:00.000+01:00"
+
+        val journey_time = TimeHelper.getJourneyTime(departure_time, arrival_time)
+
+        Assert.assertEquals("01:01", journey_time)
+    }
+
+    @Test
+    fun getJourneyTime_longTimeJourneyIsCorrect() {
+        val departure_time = "2022-07-24T14:34:00.000+01:00"
+        val arrival_time = "2022-07-26T15:35:00.000+01:00"
+
+        val journey_time = TimeHelper.getJourneyTime(departure_time, arrival_time)
+
+        Assert.assertEquals("49:01", journey_time)
+    }
+
+    @Test
+    fun getJourneyTime_invalidInput() {
+        val departure_time = "2022-07-24T14:54:00.000+01:00"
+        val arrival_time = "2022-07-24T13:35:00.000+01:00"
+
+        val journey_time = TimeHelper.getJourneyTime(departure_time, arrival_time)
+
+        Assert.assertEquals("INVALID JOURNEY TIME", journey_time)
```

Comment:
If this gets changed to an exception you can use `assertFailsWith<ExceptionType> { TimeHelper.getJourneyTime(...) }`